packages/material-ui-unstyled/src/SliderUnstyled/SliderUnstyled.js

```diff
@@ -432,13 +435,22 @@ const SliderUnstyled = React.forwardRef(function SliderUnstyled(props, ref) {
     focusThumb({ sliderRef, activeIndex, setActive });
     setValueState(newValue);
 
+    if (draggingTimer.current === null) {
+      draggingTimer.current = setTimeout(() => {
+        setDragging(true);
+      }, 0);
```

Comment:
@oliviertassinari what did setting this to 0 fix? 

It reintroduces a bug that I fixed by adding the setTimeout with a delay of 100: clicks that are a little bit draggy (imprecise clicks) will remove the animation.

https://user-images.githubusercontent.com/31566929/108099142-242b3880-707c-11eb-8385-37e414bb1a6d.mov

Reply:
@remyoudemans are you using a trackpad on macOS? Yeah, it's the downside of this change. I'm optimizing for mouse users on Windows. Try the following, set the timeout to 100, and do what a regular user would do, move the mouse right above the thumb, press the left button of the mouse, and quickly drag it. You will notice a weird initial lag.

Reply:
I see. I can often get the same result with a mouse on macOS, but have no Windows machine to test it on.

From an end-user's perspective, it might feel like the transition isn't very consistent, but that also depends on their clicking style. Hard for me to tell whether that means this is an improvement (closer to Material design) or a drawback (less consistent UX).
I've definitely been looking too close at Sliders for the last 2 weeks to have any idea what a real user might think of it 😝

Reply:
> drawback (less consistent UX)

Agree, it's my main concern too with the updated approach. I need to test with an actual mouse to see how it feels like. Hold on

Reply:
Yeah, you are right, not great 😁